README.md

```diff
@@ -106,12 +106,19 @@ First, run `make setup-sol-cli` to fetch the necessary scripts. Requires `truffl
 
 Start a ganache instance with:
 ```
-make start-eth
+cd ../chainbridge-solidity
+make start-ganache
 ```
```

Comment:
Lets just defer to the other readme: https://github.com/chainsafe/chainbridge-solidity#commands